library/src/main/java/com/instacart/library/truetime/DiskCacheClient.java

```diff
@@ -68,7 +68,9 @@ boolean isTrueTimeCachedFromAPreviousBoot() {
 
         // has boot time changed (simple check)
         boolean bootTimeChanged = SystemClock.elapsedRealtime() < getCachedDeviceUptime();
-        TrueLog.i(TAG, "---- boot time changed " + bootTimeChanged);
+        if (bootTimeChanged) {
```

Comment:
What is the value of limiting this log (also the boolean value being logged is redundant now)?